### ISSUE TITLE
Alt text for images

### ISSUE THREAD

**Issue by SamuelmdLow** [ubyssey/ubyssey-dispatch-theme#1444]
For accessibility purposes, we should have editors enter alt text describing the images they upload to the site. We could make this take less time by using image-to-text models but that might cost money for API requests.
